web-registry/src/pages/Dashboard/MyProjects/MyProjects.utils.tsx

```diff
@@ -0,0 +1,78 @@
+import { NavigateFunction } from 'react-router-dom';
+
+import {
+  useCreateProjectMutation,
+  useCreateWalletMutation,
+  WalletByAddrQuery,
+} from 'generated/graphql';
+import { UseStateSetter } from 'types/react/use-state';
+import { Wallet } from 'lib/wallet/wallet';
+
+type SubmitCreateWalletParams = {
+  wallet?: Wallet;
+  createWallet: ReturnType<typeof useCreateWalletMutation>[0];
+  setError: UseStateSetter<string | null>;
+};
+
+export async function submitCreateWallet({
```

Comment:
just a side note: this won't be needed anymore once we have keplr login, I'll create an issue to get rid of this logic